src/module.js

```diff
@@ -212,12 +212,14 @@ export default class DailyIframe extends EventEmitter {
               safari: ">=12",
               edge: "<0",
             }
-          });
+          }),
+          isScreenShareSupported = isValidBrowser && browser.satisfies({ chrome: ">0" });
```

Comment:
do we only support chrome? or do we also support chromium and electron?

Reply:
Today we only support Chrome, I believe. This is a reflection of the (outdated) check that exists here: https://github.com/kwindla/pluot-core/blob/562cdbccffc1357b3dd1c94467c252c0fed05912/js/PluotUtil.js#L832

Reply:
Side note: it's not ideal that we currently are duplicating our browser support checking in `daily-js` and `pluot-core`, but that's another discussion

Reply:
Shoot, actually just tested a basic daily.co call on Opera and Vivaldi (both of which are based on Chromium) and screen share *does* seem to work. Maybe I misread how our internal check is being done...

Reply:
Let's hold off on this PR until that's figured out. Good catch!!

Reply:
Oh I see what's going on. A bunch of browsers have the string "Chrome" in their user agent

Reply:
OK, here's what I think is happening: the *intent* of the check in `PluotUtil` is to only support Chrome or Chrome-like browsers for screen-sharing (hence the popup specifically telling you to get Chrome to do screen sharing). However, the `PluotUtil` check *also* happens to catch all Chromium-based browser (except Edge) in the process. Let's just make this PR expose `pluot-core`'s current *intended* supported browser set to any developer using `daily-js`. There's a related task to then expand the set of supported browsers.

Reply:
That sounds exactly right!

Reply:
K, updated. See the latest commit message for the rationale.